nautobot_circuit_maintenance/tests/test_graphql.py

```diff
@@ -18,7 +18,7 @@ class GraphQLTestCase(TestCase):
     """GraphQL tests."""
 
     @classmethod
-    def setUp(cls):
+    def setUp(cls):  # pylint: disable=arguments-differ
```

Comment:
remove `@classmethod`, change signature to `def setUp(self)`, pylint disable should no longer be needed.